### PR TITLE
Fix structure

### PR DESCRIPTION
**Pull request by goruha**
## What 
* Fix module structure

## Why
* Follow one pattern for all user data modules

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+}
 
-variable "github_team" {}
\ No newline at end of file
+variable "github_team" {
+  description = "GitHub Team for Membership to Grant SSH Access"
+}
+
+variable "os" {
+  default     = "ubuntu"
```
Comment (SweetOps) [inline variables.tf]: I think should be added the major version of OS or upstart-type e.g. `default     = "ubuntu-16"` or `default     = "ubuntu-systemd"`

Comment (SweetOps) [reply variables.tf]: same with `ubuntu.sh`

**Review by osterman — COMMENTED** [review]
Maybe we can add a release variable rather than overload os?

File: variables.tf (hunk 1 above)
Comment (goruha) [reply variables.tf]: I prefer release variable.
If we will follow ubuntu-16 it will be very long list.

Comment (goruha) [reply variables.tf]: also upstart system would be better then version of ubuntu.
So finally I suggest add `upstart` variable default point to `systemd`

Comment (const-bon) [reply variables.tf]: I'm leaning to `sysv` and `systemd` variables too.
We may have `ubuntu 12`, `ubuntu 14`, `ubuntu 16`, `ubuntu 18`, the same for `CentOS` and `RHEL`. Too many combinations.
With `sysv` and `systemd` we might have only four: `debian-sysv`, `debian-systemd`, `redhat-sysv`, `redhat-systemd`.

Comment (osterman) [reply variables.tf]: I think adding lots of variables for all the dimensions of complexity is a lost cause. This is why I think just having something called `flavor` is the most flexible. Can have `flavor=debian-systemd` or `flavor=ubuntu14-systemd` or `ubuntu14-systemd-no-sudo` etc

**Review by osterman — APPROVED** [review]

**Pull request merged**